packages/dbml-core/src/parse/dbml/parser.pegjs

```diff
@@ -183,6 +183,21 @@ ref_body
       settings: ref_settings
     };
   }
+//CHANGE
+RefField
+  = sp* field:(RefSingleField/RefMultipleFields) { 
```

Comment:
We didn't allow spaces before `.` so to make it consistent pls remove `sp*` here.